pycontw/pycontw-frontend
# Feat/2022 cfp

## Types of changes
* **New feature**

## Description
Need to update cfp related page for PyCon APAC 2022

## Steps to Test This Pull Request
Run the application and check the contents in
- http://localhost:3000/2021/zh-hant/speaking/cfp
- http://localhost:3000/2021/zh-hant/speaking/recording
- http://localhost:3000/2021/zh-hant/speaking/talk
- http://localhost:3000/2021/zh-hant/speaking/tutorial

is correct

## Expected behavior
The content should be the same in here
https://docs.google.com/document/d/12DgHQtNHRlMupau9XArtQ1eqF0nUrZ_K1DxqmbVPpGA/edit?usp=sharing
https://docs.google.com/document/d/1kh4SG2iOWADmtW1-C_Kd7eBnUOOeKN0hMhROHq4A3x0/edit?usp=sharing
https://docs.google.com/document/d/130l90H283YNAYk7hggwjqH91vFq6iSK8ZRaj5XSv29Q/edit?usp=sharing
https://docs.google.com/document/d/1Q3QfL9Kv1Z7hl-LwNiwtXvAR39EMTKuZttM4b5VjHfM/edit?usp=sharing
(permission required)

## Progress
- [x] Call for Proposals
- [x] Recording Release
- [x] How to Propose a Talk
- [x] How to Propose a Tutorial

## Related Issue


## Additional context
